### DISCUSSION TITLE
Better integration with Firetools/Firejail Configuration Wizard

### DISCUSSION THREAD

**Discussion by SuperSentient** [netblue30/firejail#6003] [Ideas]
I would like to see Better integration with the Firetools/Firejail Configuration Wizard such as having an option in the FJCW to have a color border around the application being sandboxed and also an option for FJ Private Home to allow you the option to allow what you want to be saved outside of PH such as bookmarks or even downloads (and you chose to allow downloads in PH when your downloading something it'll prompt you with a popup window asking do you really want to download) and then Firetools should have a section for Private Home browsers. This would make it easier for people to use PH on their favorite browser.

**Comment by rusty-snake** [comment]
Firetools repo can be found at https://github.com/netblue30/firetools

**Reply by SuperSentient** [reply]
How do I make a suggestion for FireTools?
